lib/local-links-manager/import/local_authorities_url_importer.rb

```diff
@@ -0,0 +1,56 @@
+require 'csv'
+
+module LocalLinksManager
+  module Import
+    class LocalAuthoritiesUrlImporter
```

Comment:
@jennyd , @h-lame and I a chat about this last night.  I think we're going to change the class name to fit with "URL", but not change inflections (at least for the moment).  If we have a pressing need to do so later on, we can address it then.